packages/react-core/src/components/HelperText/HelperTextItem.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/HelperText/helper-text';
+import { css } from '@patternfly/react-styles';
+import MinusIcon from '@patternfly/react-icons/dist/js/icons/minus-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-triangle-icon';
+import CheckIcon from '@patternfly/react-icons/dist/js/icons/check-icon';
+import TimesIcon from '@patternfly/react-icons/dist/js/icons/times-icon';
+
+export interface HelperTextItemProps extends React.HTMLProps<HTMLDivElement | HTMLLIElement> {
+  /** Content rendered inside the helper text item. */
+  children?: React.ReactNode;
+  /** Additional classes applied to the helper text item. */
+  className?: string;
+  /** Sets the component type of the helper text item. */
+  component?: 'div' | 'li';
+  /** Variant styling of the helper text item. */
+  variant?: 'default' | 'indeterminate' | 'warning' | 'success' | 'error';
+  /** Icon prefixing the helper text. This property will override the default icon paired with a dynamic helper text. */
+  icon?: React.ReactNode;
```

Comment:
I think `icon` makes sense here, not sure it needs the extra qualifier since both helper text cases (generic/dynamic) will set an icon if the property is passed. What do you think?

Reply:
the `Alert` component has a `customIcon` prop.  This seems like a similar use case.

Reply:
I just did a search and notification drawer just has "icon". so we are not consistent across components.

Reply:
I think we should at least add the word `custom`  to the description.